### PR TITLE
/home/root is not $HOME for root

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

**Comment by crawford** [comment]
This is correct, but can you also throw in a short example service that pulls a container? Something like:

```
[Unit]
Description=Hello World

[Service]
WorkingDirectory=/root
ExecStartPre=-/usr/bin/docker kill hello-world
ExecStartPre=-/usr/bin/docker rm -f hello-world
ExecStartPre=/usr/bin/docker pull quay.io/example/hello-world:latest
ExecStart=/usr/bin/docker run --rm --name hello-world quay.io/example/hello-world:latest
ExecStop=-/usr/bin/docker stop hello-world
```

**Comment by jzelinskie** [comment]
@crawford @robszumski PTAL

File: enterprise-registry/configure-machines/index.md [hunk 1]
```diff
@@ -66,3 +67,22 @@ If you already have images in your registry, test out a pull:
...
+[Service]
+WorkingDirectory=/root
+ExecStartPre=-/usr/bin/docker kill hello-world
+ExecStartPre=-/usr/bin/docker rm -f hello-world
+ExecStartPre=/usr/bin/docker pull quay.io/example/hello-world:latest
+ExecStart=/usr/bin/docker run --rm --name hello-world quay.io/example/hello-world:latest
+ExecStop=-/usr/bin/docker stop hello-world
+```
+
+Without setting the working directory, docker will not be able to discover the .dockercfg file and will not have the credentials to pull private images.
```
Comment (crawford) [inline enterprise-registry/configure-machines/index.md]: If the working directory is not set,

**Comment by robszumski** [comment]
LGTM

**Pull request merged**